docs/roadmap.md

```diff
@@ -0,0 +1,33 @@
+# Eclipse PASS Project Roadmap
+
+This roadmap defines the primary initiatives of the Eclipse PASS Project, organized by anticipated release.
+
+If you are interested in helping to define and/or contribute to this roadmap, please get in touch. We're always happy to welcome new project contributors!
+
+# Planned Releases
+## 0.1.0
```

Comment:
I think we should add a note about aligning all services to use the latest version of the data model